clustered/server/src/main/java/org/ehcache/clustered/server/offheap/OffHeapChainStorageEngine.java

```diff
@@ -175,6 +339,53 @@ public boolean shrink() {
     return storage.shrink();
   }
 
+  protected ByteBuffer getExtensionHeader(long chainAddress) {
```

Comment:
@chrisdennis done.. I was able to remove both chainToExtensionAddress and extensionToChainAddress.. Please take a look.